src/client/components/Registration.jsx

```diff
@@ -7,7 +7,7 @@ const emptyUser = {
     password: ""
 }
 
-const registerURL = "http://localhost:4000";
+const registerURL = "http://localhost:4000/user/register"; // need to update this URL for when we test the fetch.
```

Comment:
Nice work saving as a constant. Not entirely sure what you mean by `need to update this URL for when we test the fetch.`

Reply:
We need to get the URL from the dotenv file instead of hard coding it like this. Comment could be better.

Reply:
I understand now, yeah you're right. We'll get there, I'm sure :)